package com.np.fun;

public class CatchThrowAndFinally {

	public static void main(String[] args) throws Exception {
		try {
			//System.out.println(tryThings());
		} catch(Exception ex){
			System.out.println(ex);
		}
	
	}

}
